hardware/raspberrypi/gpio/gpio_pads_control2.md

```diff
@@ -0,0 +1,92 @@
+# GPIO pads control
+
+Extracted from BCM2835 full data sheet, from Gert van Loo 2-August-2012. Rewritten into this markdown document by Robert Bisewski in 21-October-2018.
+
+### Errata
+
+In the previous version on scribd, I accidentally dropped the password bits (bits 31..24) which need to be 5A in order to write a different value to the register!
+
```

Comment:
Does this errata section need to be here? Should be included already in the body below.